### DISCUSSION TITLE
Cannot preview images on Markdown editor

### DISCUSSION THREAD

**Discussion by yisheng** [directus/directus#16535] [Archive]
### Describe the Bug

Cannot preview images on Markdown editor, reporting `403`.

The image will be shown correctly if append `?access_token=xxx` to the url manually.

![1](https://user-images.githubusercontent.com/352986/202885354-84c67979-057b-4c1f-8d14-2c908027b112.png)

![2](https://user-images.githubusercontent.com/352986/202885358-a3e4fb2d-a5a1-49cd-9fa7-293b36224196.png)

![3](https://user-images.githubusercontent.com/352986/202885363-a5b3e2ad-504b-483a-aa1a-1f7063193722.png)

### To Reproduce

1. Add a `Markdown` field
2. Upload an image
3. Click `Preview` button

### Errors Shown

`Failed to load resource: the server responded with a status of 403`

### What version of Directus are you using?

9.20.3

### What version of Node.js are you using?

16.8.0

### What database are you using?

MySQL

### What browser are you using?

Chrome

### How are you deploying Directus?

directus.cloud

**Comment by licitdev** [comment]
You may add a static access token in the interface options for your markdown field.
Alternatively, you may enable public access to assets by allowing read permissions for the `directus_files` system collection.

### Configure token

![image](https://user-images.githubusercontent.com/26413686/202887341-c7799591-ad02-4bed-9b35-a2a7c62054fc.png)

### Token appended upon insertion

https://user-images.githubusercontent.com/26413686/202887452-15096f50-6160-42df-997e-0af4dd8303c8.mov

**Reply by yisheng** [reply]
Thanks!
